storage/mysql/mysql.go

```diff
@@ -59,10 +59,7 @@ type Storage struct {
 // New creates a new instance of the MySQL-based Storage.
 // Note that `tessera.WithCheckpointSignerVerifier()` is mandatory in the `opts` argument.
 func New(ctx context.Context, db *sql.DB, opts ...func(*tessera.StorageOptions)) (*Storage, error) {
-	opt := tessera.ResolveStorageOptions(&tessera.StorageOptions{
-		BatchMaxAge:  defaultQueueMaxAge,
-		BatchMaxSize: defaultBatchMaxSize,
-	}, opts...)
+	opt := tessera.ResolveStorageOptions(opts...)
```

Comment:
This changes the behaviour of mysql to switch to a batch size of 1 instead of 256?

Reply:
It does, but that brings it into line with the others, and I've just changed the "line" to `256` within `250ms`.

We can revisit in the future.

Reply:
The `defaultBatchMaxSize` and `defaultQueueMaxAge` consts in this file can be removed.